tests/Testcontainers.ActiveMq.Tests/ArtemisContainerTest.cs

```diff
@@ -0,0 +1,78 @@
+namespace Testcontainers.ActiveMq;
+
+public abstract class ArtemisContainerTest : IAsyncLifetime
+{
+    private readonly ArtemisContainer _artemisContainer;
+
+    private readonly string _username;
+
+    private readonly string _password;
+
+    private ArtemisContainerTest(ArtemisContainer artemisContainer, string username, string password)
+    {
+        _artemisContainer = artemisContainer;
+        _username = username;
+        _password = password;
+    }
+
+    public Task InitializeAsync()
+    {
+        return _artemisContainer.StartAsync();
+    }
+
+    public Task DisposeAsync()
+    {
+        return _artemisContainer.DisposeAsync().AsTask();
+    }
+
+    [Fact]
+    [Trait(nameof(DockerCli.DockerPlatform), nameof(DockerCli.DockerPlatform.Linux))]
+    public async Task EstablishesConnection()
+    {
+        // Given
+        var connectionFactory = new ConnectionFactory(_artemisContainer.GetBrokerAddress());
+        connectionFactory.UserName = _username;
+        connectionFactory.Password = _password;
+
+        // When
+        using var connection = await connectionFactory.CreateConnectionAsync()
+            .ConfigureAwait(false);
+
+        await connection.StartAsync()
+            .ConfigureAwait(false);
+
+        // Then
+        Assert.True(connection.IsStarted);
```

Comment:
Can we send a message to make sure the connection between client and service happen?